src/App.tsx

```diff
@@ -1,75 +1,56 @@
-    logoProps={logoProps}
-    toolbar="Toolbar"
-    avatar=" | Avatar"
-  />
-);
-
-const OldApplication: React.FC = () => {
-  const [victimList, setVictimList] = useState([]);
-  const [isDataReady, setIsDataReady] = useState(true);
-  const [isResponseOk, setIsResponseOk] = useState(true);
+import { Page, PageHeader, Flex, FlexItem } from "@patternfly/react-core";
+import Map from "./components/map/map";
+import Header from "./components/header/header";
+import Search from "./components/search/search"
+import VictimDetail from "./components/victim-details/victim-details";
+import configureStore from "./redux/store";
+import { useSelector } from "react-redux";
 
-  const fetchDetails = async (data: SearchData) => {
-    setIsDataReady(false);
-    console.log(
-      "sending request to: ",
-      process.env.REACT_APP_BACKEND_URL + `/find/victim/byName/${data.name}`
-    );
-    const response = await fetch(
-      process.env.REACT_APP_BACKEND_URL + `/find/victim/byName/${data.name}`
-    );
 
-    if (response.ok) {
-      const result = await response.json();
-      setVictimList(result.map.victims.list);
-      setIsResponseOk(true);
-    } else if (response.status === 503) {
-      setVictimList([]);
-      setIsResponseOk(false);
-      console.log("service unavailable.");
+const App: React.FC = () => {
+    // const [victimList, setVictimList] = useState();
+    let noDataFound: Boolean = false;
+    let initialDisplay: Boolean = true;
+
+    const storedState:any = useSelector((state) => state);
+
+    if(storedState.SearchReducer.name == "VICTIM_NAME"){
+        console.log(storedState.name);
+        initialDisplay = true;
+    }else if(storedState.SearchReducer.name.map.victims.list.length != 0){
+        noDataFound = true;
+        initialDisplay = false;
+
+    }else {
+        noDataFound = false;
+        initialDisplay = false;
     }
-    setIsDataReady(true);
-  };
 
-  return (
-    <Page header={Header}>
-      <SearchForm onFormSubmit={fetchDetails}></SearchForm>
-      <DisplayList
-        responseOk={isResponseOk}
-        isReady={isDataReady}
-        dataArray={victimList}
-      ></DisplayList>
-    </Page>
-  );
-};
-
-const App: React.FC = () => {
-  return (
-    <Router>
-      <Route path="/" component={OldApplication} />
-      <Route path="/newfindmyrelative" component={Homepage} />
-    </Router>
-  );
+    return (
+        <div>
+            <Header />
+            <Search />
+            {
+    !initialDisplay ? 
+                noDataFound ? 
+				<Flex className="search-bar-center">
+						<FlexItem>
+							<Map />
+						</FlexItem>
+						<FlexItem>
+							<VictimDetail />
+						</FlexItem>
+				</Flex> : 
+				<Flex>
+                        {/* Exxecute the below code if the data is empty / For NO DATA response */}
+                        <div>No Data. Please check Victim's name and try again!</div>
+                    </Flex>
+            : null 
+}
+        </div>
+    );
```

Comment:
Add the `App.tsx` changes to `Homepage.tsx` and revert all changes in this file. This will make the route function properly.

Reply:
As there was some problem with the route created, it was rendering both old and the new app, now it has been resolved.